package.json

```
@@ -3,11 +3,23 @@
   "version": "2.0.0",
   "description": "Relic Data for Warframe",
   "main": "dist/index.js",
+  "files": [
+    "dist/*",
+    "!dist/*.ts",
+    "!dist/spec/**",
+    "dist/index.d.ts"
+  ],
   "types": "dist/index.d.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "build": "tsc",
-    "start": "npm run build && node dist/build.js"
+    "coverage": "npm test && nyc report --reporter=text-lcov | coveralls",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "postinstall": "install-peerdeps @wfcd/eslint-config@^1.2.0 -S",
+    "prepack": "npm run build",
+    "start": "npm run build && node dist/build.js",
```

Wrong filename casing.
```suggestion
    "start": "npm run build && node dist/Build.js",
```